supervision/classification/core.py

```diff
@@ -67,6 +72,34 @@ def from_yolov8(cls, yolov8_results) -> Classifications:
         confidence = yolov8_results.probs.data.cpu().numpy()
         return cls(class_id=np.arange(confidence.shape[0]), confidence=confidence)
 
+    @classmethod
+    def from_ultralytics(cls, ultralytics_results) -> Classifications:
+        """
+        Creates a Classifications instance from a (https://github.com/ultralytics/ultralytics) inference result.
+
+        Args:
+            ultralytics_results (ultralytics.yolo.engine.results.Results): The output Results instance from ultralytics model
+
+        Returns:
+            Classifications: A new Classifications object.
+
+        Example:
+            ```python
+            >>> import cv2
+            >>> from ultralytics import YOLO, FastSAM, SAM, RTDETR
```

Comment:
Since `FastSAM, SAM, RTDETR` not used, let's remove that.

Reply:
Addressed :) . Good catch